ct-aa/dotTheory---TypeScript
# JS-970: Added Modules & Namespaces sample project

# Related Tickets
[JS-970](https://codeandtheory.atlassian.net/browse/JS-970)

# Description
- Added the Modules & Namespaces sample project.

# Validation
- Open the terminal.
- Run "cd modules-and-namespaces" to change the directory.
- Run "npm install".
- Run `npm run start`.
- Go to the browser. The Drag and Drop app should display and function properly.

https://github.com/ct-aa/dotTheory---TypeScript/assets/93116252/78fe0d05-bde1-4fb7-a321-dae07ff9f9b9

# Risks
N/A
